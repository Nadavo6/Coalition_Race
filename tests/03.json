{
  "parties": [
    {
      "name": "McLaren's",
      "mandates": 30,
      "join_policy": "L"
    },
    {
      "name": "Central Perk",
      "mandates": 30,
      "join_policy": "L"
    },
    {
      "name": "Ba Sing Se",
      "mandates": 30,
      "join_policy": "L"
    },
    {
      "name": "The Krusty Krab",
      "mandates": 30,
      "join_policy": "L"
    }
  ],
  "graph": [
    [0, 4, 5, 6],
    [4, 0, 7, 5],
    [5, 7, 0, 4],
    [6, 5, 4, 0]
  ],
  "agents": [
    {
      "party_id": 0,
      "selection_policy": "E"
    },
    {
      "party_id": 3,
      "selection_policy": "E"
    }
  ]
}
  