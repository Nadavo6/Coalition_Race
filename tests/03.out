[
    {
        "agents": [
            {
                "id": 0,
                "partyId": 0
            },
            {
                "id": 1,
                "partyId": 3
            }
        ],
        "coalitions": [
            {
                "mandates": 30,
                "parties": [
                    0
                ]
            },
            {
                "mandates": 30,
                "parties": [
                    3
                ]
            }
        ],
        "edges": [
            {
                "from": 0,
                "to": 1,
                "weight": 4
            },
            {
                "from": 0,
                "to": 2,
                "weight": 5
            },
            {
                "from": 0,
                "to": 3,
                "weight": 6
            },
            {
                "from": 1,
                "to": 0,
                "weight": 4
            },
            {
                "from": 1,
                "to": 2,
                "weight": 7
            },
            {
                "from": 1,
                "to": 3,
                "weight": 5
            },
            {
                "from": 2,
                "to": 0,
                "weight": 5
            },
            {
                "from": 2,
                "to": 1,
                "weight": 7
            },
            {
                "from": 2,
                "to": 3,
                "weight": 4
            },
            {
                "from": 3,
                "to": 0,
                "weight": 6
            },
            {
                "from": 3,
                "to": 1,
                "weight": 5
            },
            {
                "from": 3,
                "to": 2,
                "weight": 4
            }
        ],
        "parties": [
            {
                "id": 0,
                "mandates": 30,
                "name": "McLaren's",
                "state": "Joined"
            },
            {
                "id": 1,
                "mandates": 30,
                "name": "Central Perk",
                "state": "Waiting"
            },
            {
                "id": 2,
                "mandates": 30,
                "name": "Ba Sing Se",
                "state": "Waiting"
            },
            {
                "id": 3,
                "mandates": 30,
                "name": "The Krusty Krab",
                "state": "Joined"
            }
        ]
    },
    {
        "agents": [
            {
                "id": 0,
                "partyId": 0
            },
            {
                "id": 1,
                "partyId": 3
            }
        ],
        "coalitions": [
            {
                "mandates": 30,
                "parties": [
                    0
                ]
            },
            {
                "mandates": 30,
                "parties": [
                    3
                ]
            }
        ],
        "edges": [
            {
                "from": 0,
                "to": 1,
                "weight": 4
            },
            {
                "from": 0,
                "to": 2,
                "weight": 5
            },
            {
                "from": 0,
                "to": 3,
                "weight": 6
            },
            {
                "from": 1,
                "to": 0,
                "weight": 4
            },
            {
                "from": 1,
                "to": 2,
                "weight": 7
            },
            {
                "from": 1,
                "to": 3,
                "weight": 5
            },
            {
                "from": 2,
                "to": 0,
                "weight": 5
            },
            {
                "from": 2,
                "to": 1,
                "weight": 7
            },
            {
                "from": 2,
                "to": 3,
                "weight": 4
            },
            {
                "from": 3,
                "to": 0,
                "weight": 6
            },
            {
                "from": 3,
                "to": 1,
                "weight": 5
            },
            {
                "from": 3,
                "to": 2,
                "weight": 4
            }
        ],
        "parties": [
            {
                "id": 0,
                "mandates": 30,
                "name": "McLaren's",
                "state": "Joined"
            },
            {
                "id": 1,
                "mandates": 30,
                "name": "Central Perk",
                "state": "CollectingOffers"
            },
            {
                "id": 2,
                "mandates": 30,
                "name": "Ba Sing Se",
                "state": "CollectingOffers"
            },
            {
                "id": 3,
                "mandates": 30,
                "name": "The Krusty Krab",
                "state": "Joined"
            }
        ]
    },
    {
        "agents": [
            {
                "id": 0,
                "partyId": 0
            },
            {
                "id": 1,
                "partyId": 3
            }
        ],
        "coalitions": [
            {
                "mandates": 30,
                "parties": [
                    0
                ]
            },
            {
                "mandates": 30,
                "parties": [
                    3
                ]
            }
        ],
        "edges": [
            {
                "from": 0,
                "to": 1,
                "weight": 4
            },
            {
                "from": 0,
                "to": 2,
                "weight": 5
            },
            {
                "from": 0,
                "to": 3,
                "weight": 6
            },
            {
                "from": 1,
                "to": 0,
                "weight": 4
            },
            {
                "from": 1,
                "to": 2,
                "weight": 7
            },
            {
                "from": 1,
                "to": 3,
                "weight": 5
            },
            {
                "from": 2,
                "to": 0,
                "weight": 5
            },
            {
                "from": 2,
                "to": 1,
                "weight": 7
            },
            {
                "from": 2,
                "to": 3,
                "weight": 4
            },
            {
                "from": 3,
                "to": 0,
                "weight": 6
            },
            {
                "from": 3,
                "to": 1,
                "weight": 5
            },
            {
                "from": 3,
                "to": 2,
                "weight": 4
            }
        ],
        "parties": [
            {
                "id": 0,
                "mandates": 30,
                "name": "McLaren's",
                "state": "Joined"
            },
            {
                "id": 1,
                "mandates": 30,
                "name": "Central Perk",
                "state": "CollectingOffers"
            },
            {
                "id": 2,
                "mandates": 30,
                "name": "Ba Sing Se",
                "state": "CollectingOffers"
            },
            {
                "id": 3,
                "mandates": 30,
                "name": "The Krusty Krab",
                "state": "Joined"
            }
        ]
    },
    {
        "agents": [
            {
                "id": 0,
                "partyId": 0
            },
            {
                "id": 1,
                "partyId": 3
            }
        ],
        "coalitions": [
            {
                "mandates": 30,
                "parties": [
                    0
                ]
            },
            {
                "mandates": 30,
                "parties": [
                    3
                ]
            }
        ],
        "edges": [
            {
                "from": 0,
                "to": 1,
                "weight": 4
            },
            {
                "from": 0,
                "to": 2,
                "weight": 5
            },
            {
                "from": 0,
                "to": 3,
                "weight": 6
            },
            {
                "from": 1,
                "to": 0,
                "weight": 4
            },
            {
                "from": 1,
                "to": 2,
                "weight": 7
            },
            {
                "from": 1,
                "to": 3,
                "weight": 5
            },
            {
                "from": 2,
                "to": 0,
                "weight": 5
            },
            {
                "from": 2,
                "to": 1,
                "weight": 7
            },
            {
                "from": 2,
                "to": 3,
                "weight": 4
            },
            {
                "from": 3,
                "to": 0,
                "weight": 6
            },
            {
                "from": 3,
                "to": 1,
                "weight": 5
            },
            {
                "from": 3,
                "to": 2,
                "weight": 4
            }
        ],
        "parties": [
            {
                "id": 0,
                "mandates": 30,
                "name": "McLaren's",
                "state": "Joined"
            },
            {
                "id": 1,
                "mandates": 30,
                "name": "Central Perk",
                "state": "CollectingOffers"
            },
            {
                "id": 2,
                "mandates": 30,
                "name": "Ba Sing Se",
                "state": "CollectingOffers"
            },
            {
                "id": 3,
                "mandates": 30,
                "name": "The Krusty Krab",
                "state": "Joined"
            }
        ]
    },
    {
        "agents": [
            {
                "id": 0,
                "partyId": 0
            },
            {
                "id": 1,
                "partyId": 3
            },
            {
                "id": 2,
                "partyId": 1
            },
            {
                "id": 3,
                "partyId": 2
            }
        ],
        "coalitions": [
            {
                "mandates": 60,
                "parties": [
                    0,
                    1
                ]
            },
            {
                "mandates": 60,
                "parties": [
                    3,
                    2
                ]
            }
        ],
        "edges": [
            {
                "from": 0,
                "to": 1,
                "weight": 4
            },
            {
                "from": 0,
                "to": 2,
                "weight": 5
            },
            {
                "from": 0,
                "to": 3,
                "weight": 6
            },
            {
                "from": 1,
                "to": 0,
                "weight": 4
            },
            {
                "from": 1,
                "to": 2,
                "weight": 7
            },
            {
                "from": 1,
                "to": 3,
                "weight": 5
            },
            {
                "from": 2,
                "to": 0,
                "weight": 5
            },
            {
                "from": 2,
                "to": 1,
                "weight": 7
            },
            {
                "from": 2,
                "to": 3,
                "weight": 4
            },
            {
                "from": 3,
                "to": 0,
                "weight": 6
            },
            {
                "from": 3,
                "to": 1,
                "weight": 5
            },
            {
                "from": 3,
                "to": 2,
                "weight": 4
            }
        ],
        "parties": [
            {
                "id": 0,
                "mandates": 30,
                "name": "McLaren's",
                "state": "Joined"
            },
            {
                "id": 1,
                "mandates": 30,
                "name": "Central Perk",
                "state": "Joined"
            },
            {
                "id": 2,
                "mandates": 30,
                "name": "Ba Sing Se",
                "state": "Joined"
            },
            {
                "id": 3,
                "mandates": 30,
                "name": "The Krusty Krab",
                "state": "Joined"
            }
        ]
    }
]
